mappings/net/minecraft/client/particle/CrackParticle.mapping

```diff
@@ -10,7 +10,7 @@ CLASS dbs net/minecraft/client/particle/CrackParticle
 			ARG 2 x
 			ARG 3 y
 			ARG 5 velocityX
-	CLASS dbs$b
+	CLASS dbs$b FactorySlimeball
```

Comment:
I'd be inclined to call this `SlimeballFactory`, as it reads nicer (and same for the snowball below). Probably worth waiting to see what others think though,

Reply:
>FactorySlimeball

No.

SlimeballFactory etc

Reply:
What abt FactoryItem?